src/coinbase/address.ts

```diff
@@ -255,14 +255,20 @@ export class Address {
    * Requests faucet funds for the address.
    * Only supported on testnet networks.
    *
+   * @param asset_id - The ID of the asset to transfer from the faucet.
    * @returns The faucet transaction object.
    * @throws {InternalError} If the request does not return a transaction hash.
    * @throws {Error} If the request fails.
    */
-  public async faucet(): Promise<FaucetTransaction> {
+  public async faucet(asset_id?: string): Promise<FaucetTransaction> {
+    if (!asset_id) {
+      asset_id = "";
+    }
```

Comment:
Is this necessary? 

`requestExternalFaucetFunds` takes an optional `assetId?`. Do we need it to be explicitly set to `""`?

Reply:
removed